data-processing/tests/test_extract_definitions.py

```diff
@@ -84,6 +123,38 @@ def test_extract_nicknames_from_before_symbols():
     assert nickname1.term_text == "t"
     assert nickname1.definition_text == "timestep"
 
+# @pytest.mark.slow
+# def test_model_extracts_nickname_symbol_separated_by_colon(model: DefinitionDetectionModel):
+#     prediction_type = 'SYM-NICK'
+#     features = model.featurize("The agent acts with a policy : SYMBOL")
+#     intents, slots, _ = model.predict_batch([features], prediction_type)
+#     assert intents[0]
+#     assert slots[0] == ["O", "O", "O", "O", "O", "DEF", "O", "TERM"]
```

Comment:
Is this one of the buggy tests with the current model? I think for such tests, our approach should be to assign a tag to skip it, instead of commenting it out.

Reply:
These are cases where the tests currently fail. How should we handle that? Skip as you mentioned?

Reply:
Yup!